app/main.py

```
@@ -1 +1,18 @@
-# write your code here
+def copy_file(command: str) -> None:
+    check_command = command.split()
+    if len(check_command) == 3:
+        use_command = check_command[0]
+        file_use = check_command[1]
+        file_to_copy = check_command[2]
+        if file_use == file_to_copy:
+            return
```

Issue: When the source and destination files are the same, the function should print an error message instead of silently returning. According to the checklist: 'If the command is invalid or the source and destination files are the same, print an error message.' Please add an error message here.